### PR TITLE
GitLab allowed groups for login

### PR DESCRIPTION
**Pull request by Bugagazavr**
Added allowed orgs/groups to login

### CODE REVIEW

File: remote/gitlab/gitlab.go [hunk 1]
```diff
@@ -454,25 +476,6 @@ func (g *Gitlab) Oauth2Transport(r *http.Request) *oauth2.Transport {
...
-func (g *Gitlab) GetOpen() bool {
-	return g.Open
-}
-
-// return default scope for GitHub
-func (g *Gitlab) Scope() string {
-	return DefaultScope
-}
-
-func (g *Gitlab) String() string {
```
Comment (tboerger) [inline remote/gitlab/gitlab.go]: Have you removed that intentional?

Comment (Bugagazavr) [inline remote/gitlab/gitlab.go]: This is a legacy code, this method deleted from interface https://github.com/Bugagazavr/drone/blob/gitlab-group-verify/remote/remote.go

Comment (tboerger) [inline remote/gitlab/gitlab.go]: Ok

**Comment by tboerger** [comment]
I have not tested it but LGTM

**Comment by bradrydzewski** [comment]
LGTM

**Comment by gtaylor** [comment]
@Bugagazavr Can you also update the feature chart here? http://readme.drone.io/setup/remotes/gitlab/

**Pull request merged**